src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -104,18 +108,10 @@
       "kind": "framework",
       "version": "@WORKLOAD_VERSION@"
     },
-    "Microsoft.Android.Runtime.CoreCLR.35.android-arm": {
```

Comment:
This and similar changes looks to be "duplicative" of #9900; this branch should probably merge with main…